src/auslib/web/admin/views/permissions.py

```diff
@@ -28,8 +28,11 @@
 ]
 
 
-def get_users():
+@requirelogin
+def get_users(changed_by):
```

Comment:
Looking at this function, and the swagger spec for its response, they're clearly disagreeing (even before this).
This might be worth updating (and noting the new login requirement) while we're making changes?